Emby.Server.Implementations/IStartupOptions.cs

```diff
@@ -0,0 +1,55 @@
+namespace Emby.Server.Implementations.ParsedStartupOptions
```

Comment:
Make the namespace the same as the file path ie: `Emby.Server.Implementations`

Reply:
Ok